### PR TITLE
Updated alert panel styling and messaging

### PR DESCRIPTION
**Pull request by ashenden**
Dependent on this PR as well https://github.com/mesosphere/dcos-ui-plugins-private/pull/467

### CODE REVIEW

File: plugins/services/src/js/components/ServiceItemNotFound.js [hunk 1]
```diff
@@ -2,25 +2,25 @@ import {Link} from 'react-router';
...
   );
 
   return (
     <AlertPanel
-      title="Not Found"
-      footer={footer}
-      icon={icon}>
-      <p className="flush-bottom">
+      title="Service not found"
+      >
```
Comment (jfurrow) [inline plugins/services/src/js/components/ServiceItemNotFound.js]: #nit Let's put this `>` on the same line as the `title` attribute

File: plugins/services/src/js/containers/services/EmptyServiceTree.js [hunk 2]
```diff
@@ -1,33 +1,30 @@
...
   return (
     <AlertPanel
-      title="No Services Deployed"
-      footer={footer}
-      icon={icon}>
-      <p className="flush-bottom">
-        Create groups to organize your services or
-        deploy a new service.
+      title="No running services"
+      >
```
Comment (jfurrow) [inline plugins/services/src/js/containers/services/EmptyServiceTree.js]: `>` on same line as `title`

File: plugins/services/src/js/pages/services/DeploymentsTab.js [hunk 3]
```diff
@@ -306,8 +305,8 @@ class DeploymentsTab extends mixin(StoreMixin) {
   renderEmpty() {
     return (
       <AlertPanel
-        title="No Deployments"
-        icon={<Icon id="services" color="neutral" size="jumbo" />}>
+        title="No active deployments"
+        >
```
Comment (jfurrow) [inline plugins/services/src/js/pages/services/DeploymentsTab.js]: `>` ⬆️

File: plugins/services/src/js/containers/services/EmptyServiceTree.js (hunk 2 above)
Comment (ashenden) [inline plugins/services/src/js/containers/services/EmptyServiceTree.js]: Fixed

**Comment by jfurrow** [comment]
:shipit: when tests pass

**Comment by orlandohohmeier** [comment]
@ashenden @jfurrow Please don't merge any PR unless it has two labels. This PR introduced a bug that broke master and we could have caught it following our review process.

**Pull request merged**